### PR TITLE
Fix #26: Moved procfile to root directory

### LINKED ISSUE #26: Move the `Procfile` in the root folder
The folders `blob/master` make no sense, moving the Procfile in the root folder and delete those folders will be the best option.

### PR DESCRIPTION
**Pull request by divyanshkul**
Made the following changes:
* Moved the Procfile to the root directory.
* Also updated the links to the Procfile in the readme.

Addresses Issue: https://github.com/jboss-outreach/facebook-meme-bot/issues/26

### CODE REVIEW

**Comment by divyanshkul** [comment]
Hey @yashovardhanagrawal had a doubt in the readme link, when we write ```../Procfile``` does this search for the file in the entire repository? (If so then I don't think my readme link change applies) 🤔

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ This project shows how to deploy a facebook meme bot Vert.x 3 applications. The
 * Using the maven plugin to heroku
 * Using the maven and ngrok
 
-If you use a fat-jar then deploying on heroku is as simple as one click. The only requirement is to create the Heroku specific [Procfile](../Procfile) with a `Dyno` of type web.
+If you use a fat-jar then deploying on heroku is as simple as one click. The only requirement is to create the Heroku specific [Procfile](https://github.com/jboss-outreach/facebook-meme-bot/blob/master/Procfile) with a `Dyno` of type web.
```
Comment (yashovardhanagrawal) [inline README.md]: make this `./Procfile`

and make the other instance too as relative link

Comment (divyanshkul) [reply README.md]: Done!

**Comment by divyanshkul** [comment]
Okay got it. Thanks @yashovardhanagrawal :)

**Comment by divyanshkul** [comment]
Hey @yashovardhanagrawal just to be sure about the relative linking, the reference point that we are talking about is the location of the readme file right? Like ```../``` moves one directory backwards from where the readme is situated, isn't it?

**Comment by yashovardhanagrawal** [comment]
Yeah right 😄

**Comment by divyanshkul** [comment]
Okay understood, thanks! @yashovardhanagrawal 😄

**Comment by divyanshkul** [comment]
Done! 😅 😉

**Pull request merged**